packages/dropdowns/src/styled/menu/StyledMenu.tsx

```diff
@@ -170,13 +206,41 @@ export interface IStyledMenuProps extends HTMLAttributes<HTMLUListElement> {
   isHidden?: boolean;
   hasArrow?: boolean;
   maxHeight?: string;
+  zIndex?: number;
+  wrapperStyle?: CSSProperties;
 }
 
-export const StyledMenu = React.forwardRef<HTMLUListElement, IStyledMenuProps>(
-  ({ hasArrow, placement, maxHeight, children, ...other }, ref) => {
+export const StyledMenu = React.forwardRef<HTMLDivElement, IStyledMenuProps>(
+  (
+    {
+      hasArrow,
+      placement,
+      maxHeight,
+      children,
+      wrapperStyle,
+      isHidden,
+      zIndex,
+      isAnimated,
+      ...other
+    },
+    ref
+  ) => {
     return (
-      <StyledMenuWrapper hasArrow={hasArrow} placement={placement}>
-        <StyledMenuView ref={ref} hasArrow={hasArrow} placement={placement} {...other}>
+      <StyledMenuWrapper
+        ref={ref}
+        hasArrow={hasArrow}
+        placement={placement}
+        style={wrapperStyle}
+        isHidden={isHidden}
+        isAnimated={isAnimated}
+        zIndex={zIndex}
+      >
+        <StyledMenuView
+          hasArrow={hasArrow}
+          placement={placement}
+          isAnimated={isAnimated}
+          {...other}
+        >
           <StyledMaxHeightWrapper maxHeight={maxHeight}>{children}</StyledMaxHeightWrapper>
```

Comment:
This is broken HTML as it results in `ul` > `div` > `li`.

Reply:
With the old PopperJS positioning logic we had to define the overflow container. I'm surprised AXE never warned about this? It's possible the new `role` values made it semantically correct?

Either way, I've moved the `maxHeight` styling logic to `StyledMenu` and removed the `<div>`

Reply:
This actually opened up a bigger can of worms than I was expecting. Modifying the DOM structure here is changing how Popper is able to calculate an accurate position while our animations are happening. Some additional complexity is that our arrow styling requires that any `overflow` values be applied to a nested element, not the Menu or it's parent element.

As a temporary fix I have made all nodes `<div>`. Even though semantically this isn't the best, it is a valid DOM structure that isn't throwing errors in AXE since Downshift is apply `role`s to all elements.

This change is complex enough that I think we should solve it with our other a11y questions in the follow-on PR.